.github/workflows/compat.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ts: ['4.7', '4.8', '4.9', '5.0', '5.1', '5.2', '5.3', '5.4']
+        ts: ['4.7', '4.8', '4.9', '5.0', '5.1', '5.2', '5.3', '5.4', '5.5']
```

Comment:
As a rule of thumb, adding a new version to the matrix means dropping the left-most one (4.7). Let me check if 4.7 is EOL... 
![image](https://github.com/mswjs/msw/assets/14984911/97eebd2d-f416-4c08-9750-3e9b7b998640)

Yeah, 4.7 is EOL now. @KaiSpencer, could you please drop 4.7 from the matrix? Thanks!